modules/samples/producers/websocket/src/main/java/org/wso2/carbon/sample/websocket/Util.java

```diff
@@ -38,7 +37,7 @@
 	 * @param filePath     Text file to be read
 	 * @param sampleNumber Number of the http sample
 	 */
-	public static String getMessageFilePath(String sampleNumber, String filePath, String url) throws Exception {
+	public static String getMessageFilePath(String sampleNumber, String filePath) throws Exception {
 		if (sampleNumber != null && sampleNumber.length() == 0) {
```

Comment:
@suhothayan Thanks for pointing this out. I've made the changes we discussed offline.